### ISSUE TITLE
"Connection refused" add a warning or notification

### ISSUE THREAD

**Issue by Mte90** [W3EDGE/w3-total-cache#505]
Right now as example if you have Redis that don't answer you don't get any notification from W3 Total Cache.
You can see if redis or memcached crashed only in the php log that are filled with stuff:

```
[17-May-2022 08:36:04 UTC] Connection refused
[17-May-2022 08:36:04 UTC] Connection refused
[17-May-2022 08:36:04 UTC] Connection refused
[17-May-2022 08:36:04 UTC] Connection refused
[17-May-2022 08:36:04 UTC] Connection refused
```

Creating an enormous log but you don't get any alert in the dashboard but W3TC usually does notification for issues like when a plugin is updated.
It is possible to get a notification also by email instead of all those in the log just once?
Maybe that disable that caching system to retry it in 5 minutes and not multiple time in every second (I guess at every new page).

**Comment by mavas84** [comment]
Hello @Mte90 

Thank you for reaching out and thank you for your suggestion.
We'll make sure to review and check this and provide a solution for this like Dashboard notification as you mentioned.
Once again, thanks for your suggestion!

**Comment by cssjoe** [comment]
@Mte90 The problem looks like another issue reported, where Redis did not crash; the connect function had an issue.  Please try the changes merged into our default branch in https://github.com/W3EDGE/w3-total-cache/pull/532 and let us know if the issue is resolved or not.  If not, then please reopen this issue.  The fix is staged for the 2.2.4 release.

**Comment by tkeriduN** [comment]
I'm still having this issue. W3tc admin shows connection to redis is ok, however the error log file is filled with "connection refused" lines. My error log was 6,5 gb in size just now, after only a few weeks of having redis enabled in w3tc.

Not only is the log size in itself a DoS hazard, but the fact that redis is not working despite w3tc says it does is also a problem.
